bamMergePreprocessing.wdl

```diff
@@ -321,9 +322,7 @@ task preprocessBam {
   }
 
   String workingDir = if temporaryWorkingDir == "" then "" else "~{temporaryWorkingDir}/"
-
   String baseFileName = "~{outputFileName}"
-
   String filteredFileName = if doFilter then
                             "~{baseFileName}~{filterSuffix}"
                            else
```

Comment:
Question bellow for the filenames and file paths: 

in `String filteredFilePath = if doMarkDuplicates then` 
shouldn't it be `if doFilter then`
and 
String `markDuplicatesFileName = if doMarkDuplicates then` shouldn't it be if doMarkDuplicates and doFilter?

Reply:
filteredFilePath is used in line 390, 395, which is used to provide ourput prefix for the case when "doMarkDuplicates" is not true. The logic is quite complcated due to originally the wdl intended to be more genearal and allow more configrations. Just copied from there if in case we need it later and keep it won't affect function.